.gitignore

```diff
@@ -19,3 +19,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+<<<<<<< HEAD
```

Comment:
fixed in my branch

Reply:
Ok.  If we merge this first, then your branch should fix it.

Reply:
yes